TTCN3_Project/CANFilter/com/testingtech/playits/canfilter/Car2XEntry.java

```diff
@@ -0,0 +1,82 @@
+package com.testingtech.playits.canfilter;
+
+public class Car2XEntry {
+  // Attributes
+  private String obd2key = "0D"; // default value
+  private String valueA;
+  private String valueB;
+  private long timestamp;
+
+  /**
+   * Backup key for communicating with ELM327
+   * @return hexadecimal obd-2 key
+   */
+  public String getObd2key() {
+    return obd2key;
+  }
+
+  /**
+   * Date in millis according to the last time 
+   * valueA and valueB where updated.
+   * @return a timestamp
+   */
+  public long getTimestamp() {
+    return timestamp;
+  }
+
+  /**
+   * Each key has a valueA, respresented as String
+   * @return first value of a given key
+   */
+  public String getValueA() {
+    return valueA;
+  }
+
+  /**
+   * Optional value representing true or false in case of an event
+   * @return true or false coded as String
```

Comment:
What is the point in coding a boolean variable as String? Please change it to boolean.